containers.go

```diff
@@ -0,0 +1,81 @@
+package docker
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+// Containers is a structure with all docker containers functions
+type Containers struct {
+	Version string
+	Client  *client.Client
+}
+
+// SetupClient for filling in Docker client instance
+func (d *Containers) SetupClient() {
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
+	}
+	d.Client = cli
+}
```

Comment:
Maybe we should allow the user to configure this by supplying a config object as well and not strictly enforce config through env vars?

I'm thinking that if an object is supplied as the argument to the function, we should treat that as a client config object, if not - default to `FromEnv`. If we decide to go for this, it could be good if we also cover it with a unit test for each of the paths.

Reply:
So you mean the usage will look like this?
```js
import containers from 'k6/x/docker/containers';

export default function () {
  const containerId = 'YOUR_CONTAINER_ID';
  cosnt env = {
    ....
  }
  containers.setup(env)
  containers.startContainer(containerId, {attach: true});
}

```

Reply:
Yeah, exactly! 👍🏼